src/components/Article/Article.jsx

```diff
@@ -98,6 +103,15 @@ export class Article extends Component {
     e.preventDefault();
   };
 
+  SocialShare = (on) => {
```

Comment:
you can remove the brackets

Reply:
I tried to add this rule `"arrow-parens": ["error", "as-needed"]` in the eslint config file but it was removing parentheses from dispatch functions as well so I chose to leave it like that. Is there any other workaround?